src/main/java/org/opentripplanner/middleware/OtpMiddlewareMain.java

```diff
@@ -112,7 +113,7 @@ private static void initializeHttpEndpoints() throws IOException, InterruptedExc
          * _encoded_ URL e.g. http://localhost:3000/#/register which allows for greater flexibility.
          */
         spark.get("/register", (request, response) -> {
-            String route = HttpUtils.getRequiredQueryParamFromRequest(request, "route", false);
+            String route = getQueryParamFromRequest(request, "route", false);
```

Comment:
I think using static imports like this might actually be a bad practice. See [this perspective here](https://stackoverflow.com/a/421127/269834).

Reply:
What is your recommendation? That we qualify the method call with the util class? I'm fine with that, but we should define an approach for our projects generally on this. Perhaps we should create an issue and handle that in a separate project-wide PR?

Reply:
Let's discuss with the team together sometime and then if we decide we want to do this, then we can handle all the changes in another PR.